osf/models/user.py

```diff
@@ -1382,7 +1382,13 @@ def get_node_comment_timestamps(self, target_id):
         """ Returns the timestamp for when comments were last viewed on a node, file or wiki.
         """
         default_timestamp = dt.datetime(1970, 1, 1, 12, 0, 0, tzinfo=pytz.utc)
-        return self.comments_viewed_timestamp.get(target_id, default_timestamp)
+        timestamp = self.comments_viewed_timestamp.get(target_id, default_timestamp)
+        if isinstance(timestamp, dict):
+            if 'node' in timestamp:
```

Comment:
`timestamp = timestamp.values[0]` ?

Doesn't really matter. Your way is safer unless there's a `'comment'` or `'registration'` or something hiding out there.